OAuth/ResourceOwner/AbstractResourceOwner.php

```diff
@@ -302,9 +321,21 @@ protected function generateNonce()
      * Configure the option resolver
      *
      * @param OptionsResolverInterface $resolver
+     *
+     * @deprecated since 0.4, to be removed in 1.0 use setupOptionsInstead
```

Comment:
``overwrite``. And ``instead`` should be its own word, not a part of the method name